README.md

```diff
@@ -27,6 +27,10 @@ Positional arguments for the 'smoke' or 'load' mode:
 2. Server URL
 3. Username
 4. Password 
+8. MinIO URL
```

Comment:
Add en example here, especially because it is expected to not have `http://`

Reply:
Also current limitation that http(s) is not supported. (We hardcode useTLS arg in minio client to be false).